pkg/ip/ip.go

```diff
@@ -291,20 +291,25 @@ func PrefixCeil(numIPs int, multiple int) int {
 	return quotient
 }
 
-// PrefixToIps converts the given prefix to an array containing all IPs in the prefix / CIDR block.
-func PrefixToIps(prefixCidr string) ([]string, error) {
+// PrefixToIps converts the given prefix to an array containing IPs in the provided
+// prefix/CIDR block. When maxIPs is set to 0, the returned array will contain all IPs
+// in the given prefix. Otherwise, the returned array of IPs will be limited to the
+// value of maxIPs.
```

Comment:
I think it would be useful for developers to understand which IPs will get returned if they pass `maxIPs` > 0. In other words, how does this function decide which IPs to return? Is it deterministic, etc.?

Reply:
Good call. I've updated the godocs based on your feedback, PTAL.